tests/paddle/test_paddle_autolog.py

```diff
@@ -0,0 +1,194 @@
+import pytest
+import paddle
+import mlflow
+import mlflow.paddle
+
+NUM_EPOCHS = 6
+
+pytestmark = pytest.mark.large
+
+
+@pytest.fixture
+def paddle_model():
+    mlflow.paddle.autolog()
+
+    paddle.set_default_dtype("float32")
+    train_dataset = paddle.text.datasets.UCIHousing(mode="train")
+    eval_dataset = paddle.text.datasets.UCIHousing(mode="test")
+
+    class UCIHousing(paddle.nn.Layer):
+        def __init__(self):
+            super(UCIHousing, self).__init__()
+            self.fc = paddle.nn.Linear(13, 1, None)
+
+        def forward(self, input):
+            pred = self.fc(input)
+            return pred
+
+    model = paddle.Model(UCIHousing())
+    optim = paddle.optimizer.Adam(learning_rate=0.01, parameters=model.parameters())
+    model.prepare(optim, paddle.nn.MSELoss())
+
+    model.fit(train_dataset, eval_dataset, epochs=NUM_EPOCHS, batch_size=8, verbose=1)
+
+    client = mlflow.tracking.MlflowClient()
+    run = client.get_run(client.list_run_infos(experiment_id="0")[0].run_id)
+    return model, run
```

Comment:
For unit test, the model is changed to UCI housing to increase the testing speed

Reply:
How large is the UCI housing dataset?

Reply:
If we save it as a file, it is less than 50Kb

Reply:
The trainng data is 403 * 14 and training label is 403 * 1